config/config_test.go

```diff
@@ -158,6 +158,57 @@ func TestParseTerragruntJsonConfigRemoteStateFullConfig(t *testing.T) {
 	}
 }
 
+func TestParseTerragruntHclConfigRetryableErrors(t *testing.T) {
```

Comment:
Consider updating some of the other existing test cases to check that `terragruntConfig.RetryableErrors` is `nil`.